app/views/posts/index.html.erb

```diff
@@ -1 +1,24 @@
-<h1>List of all posts</h1>
\ No newline at end of file
+<h1>List of all posts</h1>
```

Comment:
- [ ] Good job so far but I noticed that your design is a bit different from the wireframe, Please make sure to implement the design from the sneak peek wireframes as required for this page for all your app pages 👍 

#### Project requirements
![Capture](https://github.com/brainskev/Blog-App/assets/6615394/82cb779a-bdd4-42ce-a532-d523fce5239b)


![blog_single_post](https://github.com/brainskev/Blog-App/assets/6615394/02e67043-dca7-4c51-b50b-278bcf550d36)
![blog_user_all_posts](https://github.com/brainskev/Blog-App/assets/6615394/1f8342aa-cae6-4bb0-b834-7f4ab78839e4)
![blog_user_page](https://github.com/brainskev/Blog-App/assets/6615394/ebbff508-d1c4-4294-938d-8ef7bf346007)
![blog_root_users](https://github.com/brainskev/Blog-App/assets/6615394/dccaaaaa-adf1-443e-9180-75183b521049)